.cbmc-batch/jobs/aws_array_list_swap/aws_array_list_swap_harness.c

```diff
@@ -0,0 +1,22 @@
+#include <aws/common/array_list.h>
+#include <proof_helpers.h>
+
+void aws_array_list_swap_harness() {
+    struct aws_array_list *list;
+    ASSUME_BOUNDED_ARRAY_LIST(list, MAX_INITIAL_ITEM_ALLOCATION, 128);
```

Comment:
We need this magical value here, because in `aws_array_list_mem_swap` function an [enum](https://github.com/awslabs/aws-c-common/blob/4de14a6e44a948d71e0ff38bad73acd335189f49/source/array_list.c#L143) is used as an upper bound in [here](https://github.com/awslabs/aws-c-common/blob/4de14a6e44a948d71e0ff38bad73acd335189f49/source/array_list.c#L152).